user-interface/src/main/java/life/qbic/datamanager/views/projects/project/info/ProjectSummaryComponent.java

```diff
@@ -447,6 +457,25 @@ private Div renderContactInfo(Contact contact) {
     var name = new Span(contact.fullName());
     var email = new Anchor("mailto:" + contact.emailAddress(), contact.emailAddress());
     contactInfo.add(name, email);
+    //Account for contacts without oidc or oidcissuer set
+    if (contact.oidc() == null || contact.oidcIssuer() == null) {
+      return contactInfo;
+    }
+    if (contact.oidcIssuer().isEmpty() || contact.oidc().isEmpty()) {
+      return contactInfo;
+    }
+    var oidcType = Arrays.stream(OidcType.values())
+        .filter(ot -> ot.getIssuer().equals(contact.oidcIssuer()))
+        .findFirst();
+    if (oidcType.isPresent()) {
+      String oidcUrl = String.format(oidcType.get().getUrl()) + contact.oidc();
+      Anchor oidcLink = new Anchor(oidcUrl, contact.oidc());
```

Comment:
The display should always be `https://orcid.org/<id>` for orcid entries. https://info.orcid.org/brand-guidelines/#h-orcid-logos-and-icons

Reply:
It doesn't have to be, the display can also be set to the [compact Orcid Id f](https://info.orcid.org/documentation/integration-guide/orcid-id-display-guidelines/)ormat to save space